interface/billing/billing_report.php

```diff
@@ -355,7 +355,9 @@ function set_button_states() {
             let curpid = await top.getSessionValue('pid');
             try {
                 if (curpid != pid) { // set patient
-                    top.clearPatient(false);
+                    if (curpid != 0) {
+                        top.clearPatient(false);
```

Comment:
I don't think this will work. I believe that the new patient will show the last/old in the Summary but name will be correct in patient bar.
Also you could move getSessionValue() to library/js/utility.js I'd hope it would be in scope anywhere header class is called.

Reply:
thanks @sjpadgett , it's working fine. Problem with moving it to `utility.js` is the lack of context for the login page so the embedded php calls will fail.